### DISCUSSION TITLE
Is there some IM platform where the discussions about tree-sitter-scala go on? Or should we(I) just use this?

### DISCUSSION THREAD

**Discussion by KaranAhlawat** [tree-sitter/tree-sitter-scala#276] [Q&A] [answered]
Pretty much what the title says 🐧

**Comment by eed3si9n** [comment, marked as answer]
Some of us hang out in different Discord servers, like the official Scala one, but given we're all spread out in the timezone I think using this is more reliable?

**Reply by KaranAhlawat** [reply]
That sounds good to me. I was playing around with the grammar to see if I could fix the `end` clause issues [here](https://github.com/tree-sitter/tree-sitter-scala/issues/275) and then I ran `tree-sitter generate` and was just wondering how long it takes for it to generate 😄

**Reply by eed3si9n** [reply]
https://github.com/tree-sitter/tree-sitter-scala/actions/runs/5147086401/jobs/9267036185 says 4 minutes on an Ubuntu machine to generate, build, and run tests.

**Reply by susliko** [reply]
@KaranAhlawat There's a trick for a faster feedback loop while developing: comment out `$.if_expression`, `$.try_expression` and `$.for_expression` from the `$.expression`, this should reduce parser generation times to a few seconds

**Comment by KaranAhlawat** [comment]
Just to update my work, it went nowhere 😅 I tried a bunch of stuff I thought might work, but was only able to make it work if I marked the `end_marker` as non-optional (which would be a bug on it's own).
